File: app/src/main/java/quangnam/com/sample/ui/test/fragment/TestPresenter.java
package quangnam.com.sample.ui.test.fragment;

import javax.inject.Inject;

import quangnam.com.sample.base.BasePresenter;
import quangnam.com.sample.data.DataManager;

/**
 * Created by quangnam on 11/23/17.
 * Project Sample
 */

class TestPresenter extends BasePresenter<ITestFragment.IView> implements ITestFragment.IPresenter {

    @Inject
    TestPresenter(DataManager dataManager) {
        super(dataManager);
    }
}

File: app/src/main/java/quangnam/com/sample/ui/test/fragment/TestPresenter.kt
package quangnam.com.sample.ui.test.fragment

import javax.inject.Inject

import quangnam.com.sample.base.BasePresenter
import quangnam.com.sample.data.DataManager

/**
 * Created by quangnam on 11/23/17.
 * Project Sample
 */

class TestPresenter @Inject
constructor(dataManager: DataManager) : BasePresenter<ITestFragment.IView>(dataManager), ITestFragment.IPresenter
